scandinavian-journal-of-rheumatology.csl

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB" page-range-format="minimal" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Scandinavian Journal of Rheumatology</title>
+    <id>http://www.zotero.org/styles/scandinavian-journal-of-rheumatology</id>
+    <link href="http://www.zotero.org/styles/scandinavian-journal-of-rheumatology" rel="self"/>
+    <link href="http://www.zotero.org/styles/annals-of-oncology" rel="template"/>
+    <link href="http://www.tandf.co.uk/journals/authors/healthcare/IRHE.pdf" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0300-9742</issn>
+    <eissn>1502-7732</eissn>
+    <updated>2016-12-26T20:54:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="short">
+        <single>ed</single>
+        <multiple>eds</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" prefix=" In: " suffix=". ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always" et-al-min="5" et-al-use-first="3" delimiter-precedes-et-al="never"/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher-place" suffix=": "/>
+    <text variable="publisher" suffix=". "/>
+  </macro>
+  <macro name="container-title">
+    <text variable="container-title" form="short" vertical-align="baseline" suffix="."/>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" suffix=". "/>
+      </if>
+      <else>
+        <text variable="title" suffix=". "/>
+      </else>
+    </choose>
+  </macro>
```

Comment:
@damnation333, the `<choose/>` doesn't do anything here anymore, since, unlike as in annals-of-oncology.csl, here the outcome will always be the same. Can you removing all the testing lines? You could even get rid of the entire "title" macro and just use `<text variable="title" suffix=". "/>` where you used to call it.